docs/questions-and-answers.md

```diff
@@ -63,3 +63,15 @@ Although it would be potentially possible, it would be inefficient to use the AE
 You can find more details <a href="/app-areas/">here</a>
 </details>
 
+<details><summary>Can you delete multiple agents at a time using `aea delete` command?</summary>
+You cannot delete multiple agents with the `aea delete` command. All the `aea` commands are currently scoped to a single project.
```

Comment:
Individual CLI calls are currently scoped to a single project. You can have multiple AEA projects in a given root directory but you will have to use the CLI for each project independently.

We are looking to add support for interacting with multiple AEA projects via a single CLI call in the future.